building/devices/versal_net.rst

```diff
@@ -0,0 +1,303 @@
+.. _versal_net:
+
+###############################
+AMD-Xilinx Versal Net VNX B2197
+###############################
+
+Instructions below show how to run OP-TEE on the Versal Net VNX B2197 development board.
+Details of the Versal Net can be found in the Versal Net Technical Reference Manual.
+
+Prerequisites
+*************
+
+This port requires proprietary binaries available upon request to AMD/Xilinx:
```

Comment:
Do you have a link to where to request this?

Reply:
> Do you have a link to where to request this?

I don't. The request has to go through AMD/Xilinx contacts first. There is no online form as far as I know.

Reply:
info@amd.com?

Reply:
Or perhaps you know who to contact if you have such a device?

Reply:
I believe you should know who to contact if you have such a device.

@nathan-menhorn Do you have any suggestion here?

Reply:
Hi @jcorbier and @jenswi-linaro these binaries are only available under NDA for this product. If you could update the wording to "This port requires proprietary binaries available upon request from your AMD-Xilinx FAE:"